lib/puppet/provider/ntp_server/command.yaml

```diff
@@ -4,13 +4,13 @@ get_values:
 get_instances:
   default: 'ntp server.+\\n'
 set_values:
-  default: '<state> ntp server <name> <key> <maxpoll> <minpoll> <prefer> <source_interface>'
+  default: '<state> ntp server <vrf> <name> <key> <maxpoll> <minpoll> <prefer> <source_interface> '
 ensure_is_state:
   default: 'true'
 attributes:
   name:
     default:
-      get_value: '.*(?:(?: server )(?:(?<name>\S*))).*'
+      get_value: '.*(?:(?: server vrf \S* | server )(?:(?<name>\S*))).*'
```

Comment:
rewrite this like so:
`'.*(?:(?: server( vrf)? )(?:(?<name>\S*))).*'` this will make `vrf` optional without worrying too much about spaces